orchestrator/index.js

```diff
@@ -168,8 +179,14 @@ contextManagerClient.init();
 
 let contextHandler = new ContextHandler(contextManagerClient);
 
-for (let i = 0; i < args.workers; i++) {
-  let exec = new Executor(contextHandler);
+//instantiate n executors, passing the executor number as a parameter,
+//to be used as a suffix of the rabbitmq queue name
+let queueNumber = config.amqp.queue_n;
+if (args.workers) {
```

Comment:
I think we can have problems if workers are not equal to queueNumber because the queue producers have been instantiated.